src/lib.rs

```diff
@@ -27,6 +27,8 @@ use crate::{
     state::{Metrics, TenantStoreArc},
     stores::tenant::DefaultTenantStore,
 };
+
+#[cfg(not(any(debug_assertions, test)))]
```

Comment:
why?

Reply:
We cannot have multiple instances of the server running with logging. It barfs about logging context if I try to do so.

Want me to add a comment?

Reply:
This disables them even just in debugging mode. Also see comment below

Reply:
I assumed this only disables when in `test`. Is there a way to do that?